workspaces/adventure-pack/package.json

```diff
@@ -59,6 +59,7 @@
     "immutability-helper": "3.1.1",
     "invariant": "2.2.4",
     "nullthrows": "1.1.1",
+    "prettier": "^3.3.3",
```

Comment:
Please remove `prettier`, as it's already in `devDependencies`. `dependencies` contains packages needed for running and `devDependencies` for developing.

Reply:
It did it automatically :/

Reply:
That's strange, it shouldn't happen. Let's revert anyway.